src/components/Appointment.js

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Form from 'react-bootstrap/Form';
+import { useNavigate } from 'react-router-dom';
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+import Button from 'react-bootstrap/Button';
+import { v4 as uuid } from 'uuid';
+import { appointmentsCreateThunk, appointmentsFetchThunk } from '../redux/slices/appointmentSlice';
+
+function Appointment() {
```

Comment:
- [ ] When I try to make an appointment with the default doctor "Dr. Andre Bile", and after filling in all the fields correctly, the submit button does not work:

![image](https://user-images.githubusercontent.com/57421823/203662055-1b00ad74-77bd-40fd-b6f9-c747a0c13c35.png)

## Explanation

This is because the doctor variables do not update unless the user actually selects a doctor.  Therefore, it is impossible to use the default selection "Dr. Andre Bile", unless you select another doctor, and then Andre.

## Tips and extra help

Force your users to actually select a doctor.  Make your default select field something like "Please select a doctor" so they have to actually select one.

That way, your doctors reservation section will work, every time. 💪